statusNotifierWatcher.js

```diff
@@ -94,7 +95,9 @@ var StatusNotifierWatcher = class AppIndicators_StatusNotifierWatcher {
 
         this._dbusImpl.emit_signal('StatusNotifierItemRegistered', GLib.Variant.new('(s)', service));
         this._nameWatcher[id] = Gio.DBus.session.watch_name(bus_name, Gio.BusNameWatcherFlags.NONE, null,
-                                                            this._itemVanished.bind(this));
+                                                            (proxy, bus_name) => {this._itemVanished(id)});
```

Comment:
Please remove the curly brackets, just use `, (proxy, bus_name) => this._itemVanished(id)})`